notebooks/tel_and_site/subsys_req_ver/m2/m2_functional_verification/LVV-T1782_M2_RMP_functionality_check_small_range.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #58.            print(f'\nFmeasured: {str(max_tangent)} \nerror_tangent: {str(tangent_force_error)} \nsum: {str(sum_force)} \nweight: {str(weight_force)} \nmax_axial: {str(max(max_axial))}')
</pre><p>You do not need the <code>str()</code> here after using the format string. You can change the digit in format as well. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>